src/index.html

```diff
@@ -16,29 +16,26 @@ <h1>Here is leader bord</h1>
     </header>
     <article>
         <div class="main-container">
+            
             <div class="score-seciont">
-                <table>
-                    <tr>
-                        <th>Name</th>
-                        <th>Score</th>
-                    </tr>
-                    <tr>
-                        <td>data</td>
-                        <td>data</td>
-                    </tr>
-                </table>
+                <button type="button" id="refresh">Refresh</button>
+                <hr>
+                <ul id="table">
+
+                </ul>
             </div>
             <div class="entry-section">
                 <form class="form">
                     <div class="name">
-                        <label class="label">Enter Name</label>
-                        <input type="text" class="input">
+                        <label class="label" >Enter Name</label>
+                        <input type="text" class="input" id="name">
                     </div>
                     <div class="score">
-                        <label class="label">Enter Score</label>
-                        <input type="text" class="input">
+                        <label class="label" >Enter Score</label>
+                        <input type="text" class="input" id="score">
```

Comment:
[OPTIONAL]

- [ ] Please validate your name and score fields so that it's impossible to submit an empty text because this will cause issues with the API. 😁
- [ ] Please also empty the name and score fields after a successful submission, this will make the user experience better! 😉

Reply:
Hey dear, Thank you for reviewing my code, but when I checked, It worked correctly. Can I send you a short video?

Reply:
Hello, you can do that yea, but still in your HTML there is no JS file import, this is wrong, so please fix that.

Reply:
Sorry for the confusion I just checked your project again, and the issue was with my npm and webpack, I tested it again it's working as expected, please submit it again, and I will approve it or another reviewer will!